docker/test/integration/resources/kafka_broker/conf/certs/server_cert.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Where is this file used? I don't see any reference to it. Also applies to its `.pem` brother.

Reply:
It's added to the server keystore, I want to keep it in the repository in case we need it in the future.